lms/djangoapps/courseware/views/views.py

```diff
@@ -111,7 +111,8 @@
 # credit and verified modes.
 REQUIREMENTS_DISPLAY_MODES = CourseMode.CREDIT_MODES + [CourseMode.VERIFIED]
 
-CertData = namedtuple("CertData", ["cert_status", "title", "msg", "download_url", "cert_web_view_url"])
+CertData = namedtuple("CertData", ["cert_status", "title", "msg", "download_url",
```

Comment:
nitpick - our convention is to break long lines using the style outlined here: http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#breaking-long-lines